modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -205,6 +195,33 @@ export async function save(
 	}
 }
 
+export async function save(
+	url: string,
+	item: unknown,
+	method: 'PUT' | 'POST' = 'PUT'
+) {
+	const response = await fetch(url, {
+		body: JSON.stringify(item),
+		headers,
+		method,
+	});
+
+	await handleResponse(response);
+}
+
+export async function saveFormData(
+	url: string,
+	formData: FormData,
+	method: 'PUT' | 'POST' = 'PUT'
+) {
+	const response = await fetch(url, {
+		body: formData,
+		method,
+	});
+
+	await handleResponse(response);
+}
+
```

Comment:
Hi @carolmariaabb. I think Milton wouldn't approve this code repetition :sweat_smile:. So you could do something like:

```suggestion javascript
export async function save(
	url: string,
	item: unknown,
	method: 'PUT' | 'POST' = 'PUT'
) {
        const isFormData = item instanceof FormData; 
	const response = await fetch(url, {
		body: isFormData ? item : JSON.stringify(item),
		...(!isFormData && {headers}),
		method,
	});

	await handleResponse(response);
}
```
Please, could you test if it works?

Reply:
It works! Thanks!